src/proxy/socks5.rs

```diff
@@ -101,14 +100,14 @@ impl Socks5 {
                                     _ = force_shutdown.changed() => {
                                         debug!(component="socks5", "connection forcefully terminated");
                                     }
-                                    _ = handle(oc, stream) => {}
+                                    _ = handle_socks_connection(oc, stream) => {}
                                 }
                                 // Mark we are done with the connection, so drain can complete
                                 drop(drain);
                                 debug!(component="socks5", dur=?start.elapsed(), "connection completed");
                             }).instrument(span);
 
-                            assertions::size_between_ref(1000, 2000, &serve);
+                            assertions::size_between_ref(1000, 3000, &serve);
```

Comment:
Did things really grow this much? Probably OK as I think socks5 is a bit more niche.